framework/src/modules/chain/defaults/genesis_block_devnet.json

```diff
@@ -0,0 +1,1760 @@
+{
```

Comment:
As of the separation of concern, the genesis block concept in current implementation is part of the application/controller layer, and passed to every module as static data option. 

So we should keep that file to application level.